ipn/ipnlocal/network-lock.go

```diff
@@ -85,13 +87,19 @@ func (b *LocalBackend) tkaFilterNetmapLocked(nm *netmap.NetworkMap) {
 //   - Everything up to date: All other cases.
 //     ∴ no action necessary.
 //
-// b.mu must be held. b.mu will be stepped out of (and back in) during network
-// RPCs.
-func (b *LocalBackend) tkaSyncIfNeededLocked(nm *netmap.NetworkMap) error {
+// tkaSyncIfNeeded will take b.tkaSyncLock, and periodically (while holding
+// b.tkaSyncLock) b.mu. Neither lock may be held when calling this function.
```

Comment:
tkaSyncIfNeeded immediately takes b.takeSyncLock which is held through out, and may take b.mu as required.

Reply:
Done.